src/Runner.php

```diff
@@ -171,6 +183,25 @@ private function checkExisting() {
 		return false;
 	}
 
+	/**
+	 * Check the Sensiolabs security component if availble.
+	 *
+	 * $security_status will be 0 if the code is beleived to be good, and
+	 * will be non-zero if vulnerabilities were detected (status == 1), or
+	 * if the vulnerability status is unknown (status == 127).
+	 */
+	protected function checkSensiolabsSecurity($composerLockPath, &$security_status) {
+		// If the security-checker app is not installed, return an empty message
+		exec( 'which security-checker.phar', $outputOfWhich, $return_code );
```

Comment:
Should we put `security-checker.phar` on `$PATH` so it's configurable based on environment?

Reply:
The `which` is only to test whether it is available; we run what is on the `$PATH`. Actually, `which` won't find it unless it's on the `$PATH`. Instructions in the README advise adding it to the path.

Reply:
Ah, right.